packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -797,10 +803,20 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                     this.pan(PAN_RIGHT);
                     break;
                 }
+                case 'Shift': {
+                    this.setState({ cursor: 'crosshair' });
```

Comment:
It should not change the cursor if zoom selection is in progress with the right mouse button being pressed.

Reply:
Fixed.